app/controllers/jobs_controller.rb

```diff
@@ -1,22 +1,10 @@
 require "cgi"
 require "uri"
+require "tango_client"
 
 class JobsController < ApplicationController
   autolab_require Rails.root.join("config", "autogradeConfig.rb")
-
-  #
-  # getRecentJobs - this function retrieves the currently running jobs
-  #
-  def getCurrentJobs
-    getJobs("0/")
-  end
-
-  #
-  # getDeadJobs - this function retrieves the recent dead jobs
-  #
-  def getDeadJobs
-    getJobs("1/")
-  end
+  include TangoClient
```

Comment:
Is this line necessary?  I'm pretty sure this defines all of the methods in TangoClient on the JobsController, which is not what we want.  Simply requiring the file should give you access to the class methods.